cmake/Platform/Windows/Packaging/PostInstallSetup.wxs

```diff
@@ -56,12 +55,6 @@
             Execute="deferred"
             Impersonate="no"/>
 
-        <CustomAction Id="ConfigurePython"
```

Comment:
If we're removing the python bootstrapping, would bundling cmake really be necessary? I believe it was only bundled to solve this specific case during the install, and then rely on the user's cmake?

If so, we could provide alternate instructions for using `winget`: https://winget.run/pkg/Kitware/CMake

Reply:
We not removing bootstrapping all together, we are removing the bootstrap process as part of the install script. The project manager will bootstrap as necessary and as the current user. Python, the venv, and all the 3rd party files are all downloaded for the current user so there is no need for root access for them in the first place